.drone.yml

```diff
@@ -0,0 +1,42 @@
+pipeline:
+  build_4:
+    image: node:4.4.2
+    commands:
+      - npm --loglevel warn install -g npm@3
+      - npm --loglevel warn install
+      - npm run test:ci
+
+  build_5:
+    image: node:5
+    commands:
+      - npm --loglevel warn install
+      - npm run test:ci
+
+  build_app:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker build -t rotm .
```

Comment:
It'll rebuild. It just sees the event as a pr even when you add extra commits so all is good with the world.